packages/rocketchat-theme/client/imports/components/table.css

```diff
@@ -105,7 +109,7 @@
 	color: var(--rc-color-primary-light);
 
 	font-size: 0.75rem;
-	font-weight: 400;
+	font-weight: 500;
 	line-height: 1rem;
 
```

Comment:
You should add `display: inline-block` and `vertical-align: middle` for all children elements of `table-fake-th` to align them vertically